platform/windows/display_server_windows.cpp

```diff
@@ -3740,8 +3740,13 @@ LRESULT DisplayServerWindows::WndProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARA
 			}
 		} break;
 		case WM_ACTIVATE: {
-			_process_activate_event(window_id, wParam, lParam);
-			return 0; // Return to the message loop.
+			// Activation can happen just after the window has been created, even before the callbacks are set.
+			// Therefore, it's safer to defer the delivery of the event.
+			if (!windows[window_id].activate_timer_id) {
+				windows[window_id].activate_timer_id = SetTimer(windows[window_id].hWnd, 1, USER_TIMER_MINIMUM, (TIMERPROC) nullptr);
+			}
+			windows[window_id].activate_state = GET_WM_ACTIVATE_STATE(wParam, lParam);
```

Comment:
I'm not sure if `GET_WM_ACTIVATE_STATE` is a standard Windows macro, I think it's defined only in WinTab API, might be better to use `LOWORD(wParam)` (which is equivalent).

Reply:
I used it because it was already there in the former code.

Besides, I see it's defined in `windowsx.h`, inside this region:
```
#pragma region Desktop Family
#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
```

Reply:
> Besides, I see it's defined in windowsx.h, inside this region:

Then it's OK.